### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #17 [November] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#9955] [Wallet and (d)App Council]
# Monthly Celo Wallet and (d)App Council

##  📆 Quick links 

- When: **Wednesday, November 9 (9am PDT/6pm CEST)**
- Where: **[Google Meet](https://meet.google.com/uwj-wfse-uzm)**
- Agenda: **comments**👇
- Previous: https://github.com/celo-org/celo-monorepo/discussions/9875
- Next: https://github.com/celo-org/celo-monorepo/discussions/10039
- Recording: after meeting

## 🤷‍♂️ What is this?

A monthly meeting for Celo **developers, founders,** **product** people (anyone really!) to come together and
- find consensus (coordinate),
- learn about needs, pain points, and desires
- present and collect technical feedback

## 🗒️  Agenda

- Comment with a short sentence describing what you'd like to discuss
- Don't worry, it can be small or big! 
- Every builder is welcome (and we are very easy going, I promise 😄)
- Upvote topics you're interested in

## Housekeeping

_You can safely ignore this!_

- [x] Send update calendar invite with Github link
- [x] Encourage additions to the agenda
- [x] Publicise on forum and Discord
- [ ] ~~RECORD meeting~~ 🎥 (we were only 3 people, so didn't record)
- [ ] ~~Share recording below and on forum~~
- [x] Create next meeting (link to next meeting above)

**Comment by fbaud** [comment]
**Continuation of discussion of interactions between WebApps and Wallets:**
PrimusMoney has submitted a project called "Words&Deed" to the _"Build With Celo" hackathon_ demonstrating the creation and exchange of NFTs in p2p mode to materialize the provision of individual services (language lessons, helping for domestic chores, renting an empty place for a party,...). 
I'd like to demonstrate the webapp to illustrate our discussion on atomic vs bulk authorizations for transactions. In advance of the call, you can find:

1. a short presentation of the project https://he-s3.s3.amazonaws.com/media/sprint/celo-hackathon/team/1564572/9d419212022_11_01_primusmoney_words___deeds.pdf
2. a link to the webapp I'd like to demo https://alfajores-webapps.primusmoney.com/my-deed/?tx=dd-0x2b31fb2d2504B2AD8B58597BCD7739FA3BBDbeDb-1&ccy=812d43b2-be77-efb7-f269-417aad0f3b17
